{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,136 +1,142 @@
 FROM python:3.6-alpine as base
-
-LABEL maintainer={{ cookiecutter.github_username }}
-
+#
 #  USAGE:
 #     cd sercices/{{ cookiecutter.project_slug }}
 #     docker build -f Dockerfile -t {{ cookiecutter.project_slug }}:prod --target production ../../
 #     docker run {{ cookiecutter.project_slug }}:prod
 #
 #  REQUIRED: context expected at ``osparc-simcore/`` folder because we need access to osparc-simcore/packages
 
-# non-root user 'scu'
-RUN adduser -D -u 8004 scu
+LABEL maintainer={{ cookiecutter.github_username }}
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+RUN adduser -D -u 8004 -s /bin/sh -h /home/scu scu
 
 RUN apk add --no-cache \
       su-exec
 
-ENV HOME /home/scu
-# TODO: Workaround for https://github.com/pypa/pip/issues/6197:
-#   - Added explicit --cache-dir that is removed at the end of the build
-#   - NOTE: After every PIP call, add rm -rf /tmp/.pipcache
-ENV PIP /home/scu/.venv/bin/pip3 --cache-dir=/tmp/.pipcache
+ENV PATH "/home/scu/.local/bin:$PATH"
```

Comment:
why this?

Reply:
After long thought, i decided to move from virtualenvs to local installation in dockers. The docker provides already the isolation of the virtualenvs and in addition it simplifies enormously the permissions in dev/production modes. 
With local instalation, we need to extend the path to include where the "service entrypoints binaries" (e.g. simcore_service_webserver, etc) are dropped ...